### PR TITLE
Post message issue

### PR DESCRIPTION
**Pull request by DaniAcu**
I tried to solve issue #25 
- First I tried to add more structure to the data to prevent randomly cut with slice(0,3). I think that we didn't know what had on this scope when is running.
- Second I prevented the use on chrome for now.

Please if you have time check it. Thanks for the opportunity to contribute :smile:

### CODE REVIEW

File: src/bridge.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default class Bridge extends EventEmitter {
       this._wall.send(event, payload, transferable);
       this._time = Date.now();
     } else {
-      this._messageQueue.push(event, payload, transferable);
+      this._messageQueue.push({ event, payload, transferable });
```
Comment (bvaughn) [inline src/bridge.js]: Can you explain why this change is actually important? Looks to me like all you're doing is wrapping args  in an object and unwrapping them below, but why?

**Comment by DaniAcu** [comment]
When use send. Before you was using a slice(0,3). And you have error with ArrayBuffer, because you continue use the values on some part apparently . I thought that you could use a object to be sure that you are sending the correct information. But I have doubts about this. I continue think and maybe the best option is clean the buffer after that you use. Thoughts? I will try to clean the buffer later

**Comment by bvaughn** [comment]
> When use send. Before you was using a slice(0,3).

`splice` was just being used to pull the next 3 values out of the queue, because [we always push three values onto the queue at a time](https://github.com/bvaughn/react-devtools-experimental/blob/a9fced9b0d4eab36f70da29a32fcaca3b2a37b6c/src/bridge.js#L37). It doesn't have anything to do with one of the values being an array.

It looks like all you're doing is putting all three values in a wrapper, and then pulling them back out again. I wouldn't expect this to have any impact (other than adding a little extra overhead).

**Comment by DaniAcu** [comment]
Mmm you right. Yep, this change it's unnecessary. Let me think more on this tricky issue. Thanks for the feedback
